extensions/product-recs/react-components/src/components/ProductRecsGallery/ProductCard.js

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+ *
+ *    Copyright 2021 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { useTranslation } from 'react-i18next';
+import { useStorefrontEvents, Price, Trigger, createProductPageUrl } from '@adobe/aem-core-cif-react-components';
+
+import classes from './ProductCard.css';
+
+const ProductCard = props => {
+    const mse = useStorefrontEvents();
+    const [t] = useTranslation();
+
+    const {
+        unit: { unitId },
+        product: { sku, name, type, productId, currency, prices, smallImage }
+    } = props;
+
+    const addToCart = (unit, product) => {
+        const { sku, type, productId } = product;
+        const { unitId } = unit;
+
+        const customEvent = new CustomEvent('aem.cif.add-to-cart', {
+            detail: [{ sku, quantity: 1, virtual: type === 'virtual' }]
+        });
+        document.dispatchEvent(customEvent);
+
+        mse && mse.publish.recsItemAddToCartClick(unitId, productId);
+    };
+
+    const renderPrice = (prices, currency) => {
+        const { minimum, maximum } = prices;
+        const isRange = !!(Math.round(minimum.final * 100) != Math.round(maximum.final * 100));
```

Comment:
Is the `!!` necessary here? the `!=` comparison should already result in a `boolean`. You may want to use a strict comparison event `!==`